### PR TITLE
finished styling all with temporary sign up and home page

### PR DESCRIPTION
**Pull request by yennycheung**

### CODE REVIEW

File: app/views/departments/_form.html.erb [hunk 1]
```diff
@@ -1,21 +1,25 @@
-<%= form_for(@department) do |f| %>
-  <% if @department.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(@department.errors.count, "error") %> prohibited this department from being saved:</h2>
+<div class="new_department row">
+  <div class="col-md-6 col-md-offset-3">
+    <h1>New Department.</h1>
```
Comment (rrshaban) [inline app/views/departments/_form.html.erb]: We shouldn't put this kind of writing inside the form. The form is used for new AND edit, which is why it was set up the way it was.

File: app/views/professors/_form.html.erb [hunk 2]
```diff
@@ -1,27 +1,29 @@
-<%= form_for(@professor) do |f| %>
-  <% if @professor.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(@professor.errors.count, "error") %> prohibited this professor from being saved:</h2>
+<div class="new_professor row">
+  <div class="col-md-6 col-md-offset-3">
+    <h1>New Professor.</h1>
```
Comment (rrshaban) [inline app/views/professors/_form.html.erb]: same thing here -- we use this form to edit as well as new.

**Comment by yennycheung** [comment]
ah okay no problem. its a quick fix

**Comment by rrshaban** [comment]
could you fix it before i merge? just push up another commit

**Comment by yennycheung** [comment]
will do now

**Pull request merged**